server/src/main/java/org/apache/iotdb/db/mpp/aggregation/Accumulator.java

```diff
@@ -18,20 +18,20 @@
  */
 package org.apache.iotdb.db.mpp.aggregation;
 
+import org.apache.iotdb.db.mpp.execution.operator.window.IWindow;
 import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
 import org.apache.iotdb.tsfile.file.metadata.statistics.Statistics;
-import org.apache.iotdb.tsfile.read.common.TimeRange;
 import org.apache.iotdb.tsfile.read.common.block.column.Column;
 import org.apache.iotdb.tsfile.read.common.block.column.ColumnBuilder;
 
 public interface Accumulator {
 
   /**
-   * Column should be like: | Time | Value |
+   * Column should be like: | ControlColumn | Value |
```

Comment:
As far as I'm concerned, this input should contain three columns
```suggestion
   * Column should be like: | ControlColumn | Time | Value |
```

Reply:
I've correct it.